### PR TITLE
wava

### PR DESCRIPTION
**Pull request by peggy-gov-bc-ca**
# Description

This PR includes the following proposed change(s):

- TCVP-741 Address Results of WAVA

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Tested locally

## Does the change impact or break the Docker build?

- [X] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/citizen-portal/nginx.conf [hunk 1]
```diff
@@ -6,13 +6,21 @@ location / {
   add_header Strict-Transport-Security "max-age=31536000; includeSubdomains; preload" always;
   add_header X-Content-Type-Options "nosniff" always;
   add_header Referrer-Policy "no-referrer-when-downgrade";
+
+  add_header Content-Security-Policy "default-src 'self' https://spt.apps.gov.bc.ca; script-src 'self' https://www2.gov.bc.ca 'unsafe-inline' 'unsafe-eval'; font-src 'self' https://*; style-src 'self' 'unsafe-inline';";
```
Comment (pbolduc) [inline src/frontend/citizen-portal/nginx.conf:10]: Can we please add a comment here on why each of these are here policies are here.

File: src/frontend/citizen-portal/nginx.conf [hunk 2]
```diff
@@ -30,3 +38,6 @@ location /nginx_status {
 location /api/ {
     proxy_pass http://citizen-api/api/;
 }
+
+server_tokens off;
```
Comment (pbolduc) [inline src/frontend/citizen-portal/nginx.conf:42]: Add a comment on what this does, copy a short description from the nginx docs if needed

**Review by pbolduc — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 3]
```diff
@@ -58,6 +58,8 @@ services:
     restart: always
     ports:
       - "8080:8080"
+    volumes:
```
Comment (pbolduc) [inline docker-compose.yml:61]: Should this just be commented out for docker compose?

**Review by pbolduc — CHANGES_REQUESTED** [review]

File: src/frontend/citizen-portal/nginx.conf (hunk 1 above)
Comment (peggy-gov-bc-ca) [reply src/frontend/citizen-portal/nginx.conf:10]: sure

File: src/frontend/citizen-portal/nginx.conf (hunk 2 above)
Comment (peggy-gov-bc-ca) [reply src/frontend/citizen-portal/nginx.conf:42]: sure

**Pull request merged**